views/js/qtiCommonRenderer/helpers/interactUtils.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * Helpers for interact library
+ *
+ * @author Christope Noël <christophe@taotesting.com>
+ */
+define([
+    'jquery',
+    'lodash',
+    'core/mouseEvent'
+], function($, _, triggerMouseEvent) {
+    'use strict';
+
+    var interactHelper = {
```

Comment:
good idea to create a helper for this, but didn't you initially planned to put this code in tao-core instead ?
it makes sense because the core/dynamicComponent then could benefit from it...